build/Common.props

```diff
@@ -10,6 +10,11 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <!--temporarily disable. See 3958-->
     <!--<AnalysisLevel>latest-All</AnalysisLevel>-->
+    <ExposeExperimentalFeatures Condition="'$(ExposeExperimentalFeatures)' == ''">true</ExposeExperimentalFeatures>
```

Comment:
Do we want to set it to true if the property is unset?

Reply:
When working in an IDE it will always be unset so this is basically driving what happens for local builds. At the moment it is `true`/`public` because I felt like local builds should have experimental things but I don't feel passionately about it.